lib/components/molecules/Form/FieldFileInput/FieldFileInput.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Input from '../../../atoms/Input';
+import Button from '../../../atoms/Button';
+import styles from './FieldFileInput.style';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  label: Node,
+  labelProps?: Object,
+  fieldProps?: Object,
+  value?: string | number,
+  placeholder?: string,
+  onChange?: Function,
+  onBlur?: Function,
+};
+
+const FieldFileInput = ({
+  id,
+  className,
+  type,
+  name,
+  label,
+  onChange,
+  onBlur,
+  labelProps,
+  fieldProps,
+  ...others
+}: Props): Node => (
+  // TODO: Add support for label position
+  <div className={className}>
```

Comment:
Does it support drag and drop based upload in a container? That was one of the req too

Reply:
Implemented.